test/src/main/configurations/HTTP/ConfigurationWebServiceListenerSender.xml

```diff
@@ -313,4 +313,40 @@
 			</pipe>
 		</pipeline>
 	</adapter>
+	
+	<Adapter name="HttpSoapSender" description="Send Http request to the TestSoapActionListener.">
+		<receiver className="nl.nn.adapterframework.receivers.GenericReceiver"
+			name="HttpSoapSender">
+			<listener className="nl.nn.adapterframework.receivers.JavaListener"
+				serviceName="ibis4test-HttpSoapSender" />
+		</receiver>
+		<Pipeline firstPipe="sendRequest">
+			<exits>
```

Comment:
I think now that we have junit tests these are no longer required. Maybe its better to remove them